src/rules/html-closing-bracket-spacing.ts

```diff
@@ -0,0 +1,127 @@
+import { createRule } from "../utils"
+import type { AST } from "svelte-eslint-parser"
+
+export default createRule("html-closing-bracket-spacing", {
+  meta: {
+    docs: {
+      description: "Require or disallow a space before tag's closing brackets",
```

Comment:
The first letter should be lowercase to be consistent with existing rules.

```suggestion
      description: "require or disallow a space before tag's closing brackets",
```